frontend/packages/shared/src/components/altinnHeader/AltinnHeader.tsx

```diff
@@ -58,7 +61,8 @@ export const AltinnHeader = ({
             </>
           )}
         </div>
-        <AltinnHeaderMenu menuItems={menuItems} />
+        {heading && <Heading className={classes.heading}>{heading}</Heading>}
```

Comment:
Maybe we can use the `size` property on the Heading component instead of the css?